src/common/dao/repository.go

```diff
@@ -183,8 +183,6 @@ func repositoryQueryConditions(query ...*models.RepositoryQuery) (string, []inte
 	}
 
 	if len(q.ProjectName) > 0 {
-		// use "like" rather than "table joining" because that
```

Comment:
Do you recall the background of this comment?
Do we need to move back to "table joining"?